stable/netsil/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+name: netsil
+version: 1.5.3
+description: Netsil Application Operations Center (AOC)
+keywords:
+  - monitoring
+  - analytics
+  - alerting
+  - metric
+home: https://netsil.com/
+icon: https://s3.amazonaws.com/docs.netsil.com/logos/Netsil_640x640-blue.jpg
+sources:
+  - https://github.com/kubernetes/charts
+maintainers:
+  - name: Matt Baldwin
```

Comment:
Please use Github username.

Reply:
Looking at other Charts, this doesn't appear to be the case. Is this a personal preference or a requirement for merge? ref. 

https://github.com/kubernetes/charts/blob/master/stable/linkerd/Chart.yaml

Reply:
This is what we started recommending now. It makes it easier to identify maintainers. Don't just take any random chart for reference.